src/OpenDebugAD7/Telemetry/DebuggerTelemetry.cs

```diff
@@ -55,6 +56,7 @@ internal sealed class DebuggerTelemetry
         public const string TelemetrySourceFileMappings = "SourceFileMappings";
         public const string TelemetryMIMode = "MIMode";
         public const string TelemetryStackFrameId = TelemetryExecuteInConsole + ".ExecuteInConsole";
+        public const string TelemetryTracepointTokens = ".Tokens";
```

Comment:
where is this being used?

Reply:
also your property shouldn't start with a `.`. I think you are missing a concatenation.